docs/SdkVersioning.md

```diff
@@ -0,0 +1,590 @@
+# SDK Versioning Strategy
+
+> :warning: THIS IS AN EXPERIMENTAL FEATURE :warning:
+
+## Version Schema
+SDK Versioning (SdkVer) is compatible with SemVer, but not following the full SemVer specification;
+
+```js
+<API>.<RELEASE>.<FIX>(-<PRERELEASE>)
+```
+
+| Item       | Description                                                  |
+| ---------- | ------------------------------------------------------------ |
+| `API`      | Version of the (public) API                                  |
+| `RELEASE`  | Release number                                               |
+| `FIX`      | Incremental number indicating incremental fixes on a release |
+
+In addition, `PRERELEASE` can follow the following two release stages:
+- [Development Releases](#development-releases): `-dev[N]`
+- [Release Candidates](#release-candidates): `-rc[N]`
+
+## Releases
+
+There are three types of Releases supported by SdkVer:
+
+- [Development Releases](#development-releases)
+- [Release Candidates](#release-candidates)
+- [Releases](#releases)
+
+### Development Releases
+
+A Development Release reflects the current state of the `main` development branch.
+The only representation of a Development Release is by the availability of a single
```

Comment:
```suggestion
The representation of a Development Release is by the availability of a single
```

leaving it open for adopters to decide what else they might deem relevant as presentation for the development releases.